Add MCP (#16)
groq/groq-changelog: CHANGELOG.md

# Groq Changelog

## 2025-09-23 (Python SDK v0.31.1, TypeScript SDK v0.32.0)

### [ADDED] Remote Model Context Protocol (MCP)

Remote Model Context Protocol (MCP) server integration is now available in Beta on GroqCloud, connecting AI models to thousands of external tools through Anthropic's open MCP standard. Developers can connect any remote MCP server to models hosted on GroqCloud, enabling faster, lower-cost AI applications with tool capabilities.

Groq's implementation is fully compatible with both the OpenAI Responses API and OpenAI remote MCP specification, allowing developers to switch from OpenAI to Groq with zero code changes while benefiting from Groq's speed and predictable costs.

**Why Remote MCP Matters:**
- **Universal interface** - Connect to thousands of remote MCP servers and tools through one open standard
- **Faster execution** - Lower round-trip latency than alternatives
- **Lower costs** - Same experiences at a fraction of the price
- **Seamless migration** - Keep your connector code, just change the endpoint

**Supported Models:**
Remote MCP is available on all models that support [tool use](https://console.groq.com/docs/tool-use), such as:

> - GPT-OSS-120B and GPT-OSS-20B

Reviewer: ```suggestion
- `openai/gpt-oss-20b`
- `openai/gpt-oss-120b`
```

> - Kimi K2 0905

Reviewer: ```suggestion
- `moonshotai/kimi-k2-instruct-0905`
```

- Qwen 3 32B
- Llama 4 Maverick and Llama 4 Scout
- Llama 3.3 70B and Llama 3.1 8B

> **Cookbook Partners:**

Reviewer: ```suggestion
**Tutorials to get started with MCP:**
```

> Get started with ready-to-use tutorials from our MCP partners:

Reviewer: ```suggestion
Learn how to easily integrate various MCP servers and their available tools, such as web search, into your applications with Groq API with these tutorials from our launch partners:
```
